.claude/hooks/block_git_no_verify.py

```
@@ -1,58 +1,168 @@
-        continue
+CONFIG_ENV_PREFIX = "--config-env="
 
-    if t == "git":
-        seen_git = True
-        sanitized.append(t)
-        continue
 
-    if seen_git and t == "commit":
-        seen_commit = True
+def _is_hooks_path(value: str) -> bool:
+    """core.hooksPath への言及か（git の設定キーは大文字小文字を区別しない）"""
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)